client/src/components/Game/FabricCanvas.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect } from 'react';
+import { FabricContext } from './FabricContextProvider';
+import { fabric } from 'fabric';
+import { saveAs } from 'file-saver';
+
+const FabricCanvas = props => {
+    const [canvas, initCanvas] = useContext(FabricContext);
+    const canvasX = 1000;
+    const canvasY = 800;
+
+    const saveCollage = () => {
+      canvas.toCanvasElement().toBlob(function(blob) {
+        saveAs(blob, props.title + '.png');
+      });
+    };
+
+    useEffect(() => {
+        const canvas = new fabric.Canvas('c');
+        initCanvas(canvas);
+
+        const title = new fabric.Textbox(props.title, {
+          fontFamily: 'sans-serif',
+          fontSize: 36,
+          top: 50,
```

Comment:
It might be good to abstract this object as well as the player textbox into their own constants.

Reply:
Could you give an example? They're already declared under `const`?